content/en/05-reference/pxl/index.md

```diff
@@ -15,23 +15,26 @@ PxL can be executed by the Pixie platform by using either the web based UI, API
 
 ## Data Types
 
-PxL has a rich type system consisting of both concrete and semantic types.
+PxL has a rich type system consisting of both concrete and semantic types. Data types are used for the following purposes:
+
+- The query planner uses types to determine which PEM agents to pull data from.
```

Comment:
this is about Semantic types, we call "concrete types" "data types" in other places. Should just call it "Semantic types"